terraform/README.md

```diff
@@ -151,13 +149,17 @@ $ terraform apply
 Once `terraform apply` completes, the private development infrastructure is deployed.
 You can now play around in your isolated copy of the infrastructure, testing and updating the changes, making sure the changes work as expected before merging the changes.
 
-## Destroying Playground Environment
+## Destroying Test Environment
 Once the configuration changes have been tested, the private development environment can be destroyed:
 ```bash
-# Destroy the private terraform worskapce
-$ ./playground/terraform-playground.sh destroy
+# Destroy the private terraform worskapce using helper script
+$ ./terraform-playground.sh destroy
+
+# Alternatively, you can use terraform command directly
+$ terraform workspace select <workspace_name>
+$ terraform apply -destroy
 ```
-The above command removes all the resources that were created for the private development environment.
+The above command(s) remove all the resources that were created for the given environment.
```

Comment:
still isn't private?

Reply:
It doesn't necessarily need to a 'private' environment, it can be any other workspace too.